app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<p id="alert">
+  <% alert %></p>
+
+<h1>Login</h1>
+<%= form_tag sessions_path do |form| %>
+
+  <div class="field">
+    <%= label_tag :username %><br>
+    <%= text_field_tag :username %>
+  </div>
+
+  <div class="field">
+    <%= label_tag :password %><br>
+    <%= password_field_tag :password %>
+  </div>
+  <br>
+  <div class="actions">
+    <%= submit_tag "Login" %>
+  </div>
+
+<% end %>
```

Comment:
@JsMontesR As a good practice tip: keep a blank space at the end of the files

Reply:
I'll keep in mind! 👌🏽 , btw, why?